scripts/configure.py

```diff
@@ -38,6 +38,16 @@
   'internet_radio' : {
     'apt' : [ 'vlc' ]
   },
+  'fmradio' : {
+    'apt' : [ 'rtl-sdr', 'git', 'build-essential', 'autoconf', 'libsndfile1-dev', 'libliquid-dev' ],
+    'script' : [
+      'cd /tmp',
+      'git clone https://github.com/windytan/redsea.git',
```

Comment:
Maybe reduce the network load of the git clone?
```suggestion
      'git clone --depth=1 https://github.com/windytan/redsea.git',
```